test/sparse_operator.jl

```diff
@@ -0,0 +1,37 @@
+using Snowflake
+using Test
+
```

Comment:
add standardized test for all AbstractOperators:

```
include("test_functions.jl")

test_operator_implementation(SparseOperator,dim=2,label="SparseOperator",values=[1.,2.,0.,3.])
 
```

Reply:
We will do that in another pull request. Let's start a new issue to address this.